core/src/test/scala/org/apache/spark/util/AccumulatorV2Suite.scala

```diff
@@ -86,4 +86,48 @@ class AccumulatorV2Suite extends SparkFunSuite {
     assert(acc.sum == 5.0)
     assert(acc.avg == 1.25)
   }
+
+  test("ListAccumulator") {
+    val maxI = 1000
+    for (nThreads <- List(1, 10)) {
```

Comment:
why do we need this? can we just do some basic unit test similar to other test cases.